apidoc/Titanium/Calendar/Event.yml

```diff
@@ -3,8 +3,8 @@ name: Titanium.Calendar.Event
 summary: An object that represents a single event in a calendar.
 description: |
     The API supports retrieving information about existing events and creating new events.  
-    Be aware that modifying or deleting existing events and creating recurring rules for events 
-    are only supported on iOS.
+    Be aware that modifying, with the exception of adding a new recurrence rule, or deleting
```

Comment:
This sentence feels clunky and is difficult to easily understand at first glance. Perhaps @jawa9000 can help massage the updated text here?

Reply:
I was trying to be as minimal as possible. But now that I think of it, a more detailed description may be needed here.